docs/DEVELOPMENT.md

```
@@ -0,0 +1,478 @@
+# Deploy with custom values
+helm install ai-studio helmcharts/ai-studio/ -f custom-values.yaml
+
+# Update deployment
+helm upgrade ai-studio helmcharts/ai-studio/ --set frontend.image.tag=latest
+```
+
+### Reorganized Azure DevOps Structure
+
+Azure DevOps pipelines have been reorganized for better maintainability:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)